AWSAuthSDK/Sources/AWSAuthCore/AWSAppleSignIn/AWSAppleSignInButton.m

```diff
@@ -51,19 +52,13 @@ - (void)commonInit {
 }
 
 - (void)createAppleButton {
-    if (NSClassFromString(@"ASAuthorizationAppleIDButton")) {
-        // Do something
-        _appleButton = [[NSClassFromString(@"ASAuthorizationAppleIDButton") alloc] init];
-        [self.appleButton addTarget:self
-                             action:@selector(logInWithProvider:)
-                   forControlEvents:UIControlEventTouchUpInside];
-        CGRect buttonFrame = self.appleButton.frame;
+    if (@available(iOS 13, *)) {
+        _appleButton = [[ASAuthorizationAppleIDButton alloc] init];
```

Comment:
If we don't use `_appleButton` outside this method, it doesn't need to be a property or ivar. Just make it a local var. (I may be missing where it's used, though)